apps/profile/views.py

```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+def show_me(request):
+    respo = {'username': request.user.username, 'email': request.user.mail, 'company': request.user.company,
```

Comment:
Der User ist immer automatisch in den Templates verfügbar mit "{ user.xyz } " also macht es kein sinn hier nochmal ein extra Kontext für Userdaten dem Template zu geben.